### DISCUSSION TITLE
Not smoothing motor

### DISCUSSION THREAD

**Discussion by posssson** [doudar/SmartSpin2k#280] [Q&A]
Hi everyone,
I come back for an issue I have. I updated the code with the develop code and now with the lowest values for the settings (shift step 50 and 600mA) the motor is way too fast
 I tried one run with 200 watts during 1 hour and when I was above 230 watts the resistance decreased too quickly and I was at 150 watts and then 2 seconds after the motor increased power again. above 230 w.. and I was stuck in a loop between 150 and 230 w and it  was hard or almost impossible to go back to 200 watts.

**Comment by MarkusSchneider** [comment]
You are talking about ERG Mode?

What have you set for ERG Sensitivity? With ERGSensitivity you have multiplier for the target incline. 
Shifter Steps only used for smaller changes or to limit the maximum change:
[see BLE_Server.cpp](https://github.com/doudar/SmartSpin2k/blob/develop/src/BLE_Server.cpp#L244)

**Reply by posssson** [reply]
Indeend in erg mode
Everything is the lowest 
Erg sensitivity = 1

**Reply by posssson** [reply]
I have assioma power tap with zwift

**Reply by MarkusSchneider** [reply]
Please change ERG Sensitivity to a value lower than 1.0 values between 0.5 and 5 are allowed (as far as I know).

**Reply by doudar** [reply]
We will have yet another totally revamped ERG mode coming soon as well. @MarkusSchneider and I have been working very hard on it. 

It’s going to be great.

**Reply by doudar** [reply]
You may need to update the html files (/data folder) as well if you manually updated the firmware. The older html didn’t let you set lower than 1 for the ERG sensitivity.

**Reply by MarkusSchneider** [reply]
You can use the "Update Firmware" Dialog and update the [settings.html](https://github.com/doudar/SmartSpin2k/blob/develop/data/settings.html).

**Reply by posssson** [reply]
Indeed with erg 0.5 it is better. I read your new ERG mode code, and indeed it will be better. I will try to understand all codes but I don't have so much time to participate (I work as control/command in satellite so I have a great experience for this sort of problems).
And thank you for your work.

**Reply by posssson** [reply]
I checkout(ed) develop i don't use update firmware.

**Comment by doudar** [comment]
@posssson - Check out the newest release 1.12.26 . It has a brand new ERG mode that learns your bike as you ride it. I think you'll like it.
